modules/monitoring-listing-user-modified-default-alerting-rules.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="listing-user-modified-default-alerting-rules_{context}"]
+= Listing user-modified default alerting rules
+
+As a cluster administrator, you can list user-modified default alerting rules for core {product-title} platform monitoring.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+* To list all user-modified default alerting rules in the `<openshift-monitoring>` namespace, run the following:
```

Comment:
To list all user-modified alerting rules in the `openshift-monitoring` namespace, run the following:

Reply:
fixed